src/medVtkInria/vtkDataManagement/vtkMetaDataSet.cxx

```diff
@@ -730,6 +730,33 @@ double* vtkMetaDataSet::GetCurrentScalarRange()
   return val;
 }
 
+double* vtkMetaDataSet::GetCurrentScalarRange(const QString & attributeName)
+{
+  double* val = new double[2];
+  val[0] = VTK_DOUBLE_MAX;
+  val[1] = VTK_DOUBLE_MIN;
+
+  if (this->GetDataSet())
+  {
+      if (this->GetDataSet()->GetPointData()->HasArray(qPrintable(attributeName)))
```

Comment:
A small thing: https://doc-snapshots.qt.io/qt5-dev/qtglobal.html#qPrintable if the `attributeName` is in UTF8 it could be better to use `qUtf8Printable() `

Reply:
It was introduced in Qt 5.4. We can't use for now :-(